docs/design/2020-06-24-placement-rules-in-sql.md

```diff
@@ -64,39 +64,16 @@ This proposal makes some intentional decisions so that both categories of use-ca
 
 ### New Syntax Overview
 
-There are two ways to specify placement rules:
-
-1. By assigning placement directly on a database, table or partition (direct assignment)
-2. By creating a new `PLACEMENT POLICY` and then applying the placement policy to a database, table or partition (placement policy)
-
-Using a `PLACEMENT POLICY` will be recommended for compliance requirements, since it can allow administrators to better keep track of usage. This can be seen as similar to how complex environments will use `ROLES` for management instead of directly assigning privileges to users.
-
-Both syntaxes are considered [`table_option`](https://dev.mysql.com/doc/refman/8.0/en/alter-table.html)s, and available in both `CREATE TABLE` and `ALTER TABLE` contexts.
-
-The two methods **are mutually exclusive**. Specifying both in a `CREATE`/`ALTER` statement will result in an error. Specifying a `PLACEMENT POLICY` on a table with direct assignment options will clear those options.
-
-#### Direct Assignment
-
-Creating a new table with directly assigned constraints. The leader is in `us-east-1` region, the followers are in `us-east-1` and `us-east-2`:
+The new syntax is based on a `PLACEMENT POLICY` statement and the ability to assign a placement policy to a schema, table or partition:
 
 ```sql
-CREATE TABLE t1 (
-	id INT NOT NULL PRIMARY KEY,
-	b VARCHAR(100)
-) PRIMARY_REGION="us-east-1" REGIONS="us-east-1,us-east-2";
+CREATE PLACEMENT POLICY policyName [placementOptions];
+CREATE TABLE t1 (a INT) PLACEMENT POLICY=policyName;
 ```
 
-In this context, "REGION" and "REGIONS" are syntactic sugar which map to the label `region`. The following labels have special reserved words (the plural is used in contexts such as followers where multiple is possible):
-- `host` and `hosts`: expected to be the same physical machine or hypervisor.
-- `rack` and `racks`: similar to host; a group of machines that are physically close together and may suffer from many of the same failures.
-- `zone` and `zones`: similar to an AWS zone; much larger degree of blast radius isolation from a rack, but still vulnerable to issues such as a natural disaster.
-- `region` and `regions`: expected to be distributed far enough apart that there is isolation from disasters.
-
-To use additional labels not in this list, see "Advanced Placement" below.
```

Comment:
Seems that we somewhat did not tell much about the sugar syntax after removing this section...

Reply:
I agree - it looks like I removed a bit too much. I'll add it back.